model/common.go

```diff
@@ -1,13 +1,13 @@
 package model
 
 type Swagger struct {
-	Swagger string `json:"swagger"`
-	Info    Info   `json:"info"`
-	Host    string `json:"host"`
-	BasePath string `json:"basePath,omitempty"`
-	Schemes []string `json:"schemes,omitempty"`
-	Paths   map[string]*PathItem `json:"paths"` // Ignoring extensions for now
-	Definitions map[string]*Schema `json:"definitions"`
+	Swagger     string               `json:"swagger"`
+	Info        Info                 `json:"info"`
+	Host        string               `json:"host"`
+	BasePath    string               `json:"basePath,omitempty"`
+	Schemes     []string             `json:"schemes,omitempty"`
+	Paths       map[string]*PathItem `json:"paths"` // Ignoring extensions for now
```

Comment:
maps do not preserve the order of items. The order of items is only important for comparing tests results ; having stable output. for that reason, this package generates special types for maps such as pattern_orderdmap